spec/requests/categories_controller_spec.rb

```diff
@@ -19,6 +19,15 @@
     expect(Category.can_vote?(category.id)).to eq(true)
   end
 
+  it "does not recreate database record" do
+    category_setting = DiscourseVoting::CategorySetting.create!(category: category)
+
+    put "/categories/#{category.id}.json", params: {
+      custom_fields: { "enable_topic_voting" => true }
+    }
+    expect(DiscourseVoting::CategorySetting.last.id).to eq(category_setting.id)
+  end
+
```

Comment:
There were two conditions modified above, should there be two corresponding test changes?

Reply:
The test I added verifies both the conditions and fails if any of them is not met.